OpenTranslator/Controllers/Awesome/PascalTranslationController.cs

```diff
@@ -272,33 +274,33 @@ void LoadFromReader(TextReader reader, string input, string filename = "")
                     switch (type)
                     {   
                         case BlockType.Id:
-                            originalText = val;
+                            translatedText = val;
                             textId = val.RemoveNonAlphanumerics().ConvertCaseString(StringExtension.Case.CamelCase).Replace(" ", String.Empty);
-                            if (String.IsNullOrEmpty(textId) && String.IsNullOrEmpty(originalText))
-                            {
-                                continue;
-                            }
-                            if (input.Equals("Import"))
-                            {
-                                InsterRecord(textId, originalText, originalText, _defaultlanguage);
-                            }
-                            GetTranslationRecord(textId, originalText, originalText, _defaultlanguage);
+                            //if (String.IsNullOrEmpty(textId) && String.IsNullOrEmpty(originalText))
```

Comment:
I need to explain this code I made:
the lines you commented were saving TextId with the english word (the same text id word) as a translated_text, this assures us that the word in english is saved in DB, for example it will save
id: theBeer, translated_text: The Beer, language: en
And after that, (in BlockType.Str block) the real translated text will be saved, in its corresponding language, for example, for spanish po file:
id: theBeer, translated_text: La Cerveza, language: es
Could you please explain how english word, and its translation is saving right now without this lines?

Reply:
I have commented the line "GetTranslationRecord" because it causes the export result it appends one string twice.
also I have commented your code because I am thinking we are going to import only English text so when 
It will come to BlockType str then I have inserted the record.
For English Language I am considering -
#: stringsunit.rseight - TextId
msgid "Eight" Text
msgstr ""